spring/jta/pom.xml

```diff
@@ -127,6 +122,22 @@
             <version>${version.org.jboss.spec.javax.transaction}</version>
         </dependency>
         <!-- Other dependencies -->
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
+            <version>2.1.1</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
+                    <artifactId>commons-logging</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-pool2</artifactId>
+            <version>2.4.3</version>
```

Comment:
would not be better to use `${..}` configured in `pom.xml` `<configuration` section, than directly define the version here?